pipeline/src/node_properties/model_inference_properties.rs

```diff
@@ -16,4 +17,12 @@ pub struct ModelInferenceRuntime {
     /// and generate a config.
     // TODO: replace url::Url with fs::PathBuf
     pub config: Option<url::Url>,
+
+    /// The ID of the inference model.
+    ///
+    // FIXME:
+    // 1. Should this really be Optional? Seems we define all runtime props as options but are they
+    //    optional or should the runtime field in the parent Properties structure be optional?
+    // 2. Is this really a runtime(only) property? If not, should be in the parent struct.
+    pub model_id: Option<Uuid>,
```

Comment:
Please have a look [here](https://www.notion.so/PIPELINE-5c9e100b5222472883b2cb47b5eb3d4d?p=45cfe47b29114f5188c621a5e6d1ba67)

I think that `model_id` is required and should be in parent struct.

Reply:
roger. BTW, that doc needs changes especially after these PRs.